cloudigrade/api/authentication.py

```diff
@@ -13,7 +13,7 @@
 logger = logging.getLogger(__name__)
 
 
-def parse_requests_header(request):
+def parse_requests_header(self, request):
```

Comment:
Although it isn't a reserved word, strongly by convention `self` is only used class instance methods to refer to the instance. Since this is a class-less function, this should be named something else more descriptive of the incoming object type.